### DISCUSSION TITLE
発表スライド進捗

### DISCUSSION THREAD

**Discussion by ritsuxis** [ritsuxis/SoftwareEcosystem-Group1#6] [General]
スライドは授業のフォーラムの方に上げています

**Comment by celorind** [comment]
こちらにもメモ

発表用資料
https://docs.google.com/presentation/d/1w_N5w5LXwlMHgHjnhCJDKm8GA3_nX_xQckFGGN0WsLQ/edit?usp=sharing

これを編集してください
信州大学のアカウントで入れます

**Comment by YSHRK** [comment]
スター数上位100個のリポジトリを調査した結果
https://docs.google.com/spreadsheets/d/175JTHrzI98IJnepLs72S3ccAluUVylyl/edit#gid=891946848

**Reply by kuuchan-code** [reply]
ありがとうございます！

**Comment by ritsuxis** [comment]
このリポジトリにRQ２用のデータを入れました。
実行したファイルも一緒に入れています。

取ったデータは「スター数上位１００個のリポジトリのIssueとPRの中で”rename”, "master", "branch"」が入っているもののタイトルとurlです。
機械的に取得したものなので、精査はしてないです。

**Reply by ritsuxis** [reply]
いくつか見てみましたけど、ちゃんとIssueなりPRで上がっているリポジトリであればマージされている気がします
議論はあんまりないかもですね

**Comment by ritsuxis** [comment]
データから実際にissueが上がっているものを調べました（RQ1の表をコピーさせてもらいました)

https://docs.google.com/spreadsheets/d/1hfEEH7ZAOI3A4-b2NrEsIUVkj9pJExXN/edit?usp=sharing&ouid=101693179644700543890&rtpof=true&sd=true

mainになっているものの半分ぐらいがissueが上がっていて、上がった時期が2020/06月頃が多いです。
これはジョージ・フロイドの死が2020/05/25だからかなと。

また、ghost(アカウント削除)されている人のissueもその中でいくつか見つけました。ほぼ全員が同じ日2020/06/15に上げていて、同じようなリンクを貼っているので多分同一人物ではないかなと思います

**Reply by ritsuxis** [reply]
スライドにRQ2の内容を入れました。
他の部分誰かお願いします。データはこのリポジトリとディスカッションに上がっているので、足りなさそうな部分があればRQ2も適宜編集してもらえるとありがたいです。
